### DISCUSSION TITLE
Tables...Am I missing something

### DISCUSSION THREAD

**Discussion by sheldonj84** [nvim-neorg/neorg#1399] [Q&A]
The Neorg table format seems very hard to read in plain text format... 

What is the thought process here, do you use them just so they can export to other docs looking nice or should Neorg make them look nice and my config isn't set up correctly.

any tips on how others use them would be appreciated. 

Thanks,

**Comment by glyh** [comment]
Also it's not even in `help neorg`. Don't know what's happening here.

**Comment by pysan3** [comment]
https://github.com/pysan3/Norg-Tutorial/blob/main/norg_tutorial.md#table-syntax

**Reply by glyh** [reply]
I am aware of this, but I get no syntax highlight. Is this expected? For now I just embed a piece of markdown code, which feels illegal lol.

**Reply by sheldonj84** [reply]
I see how the syntax works, but reading it as a note in Neovim is not something my brain can easily do. Tables are meant to put data in a clean organized way so i can better understand what I am looking at. Having to calculate the offset while looking at my notes is fairly difficult. 

is there a way to see  it in Neovim, or is this just purely for exporting to other docs?

**Reply by pysan3** [reply]
As for the rich table syntax there are no support from **neorg** right now. It is just that the syntax is defined in the spec (so it will surely be a _valid_ syntax) and the tree-sitter parser is able to parse it, and that's all we have right now. Jfyi norg is the file format (specification) and neorg is the plugin for QoL like highlighting, keybinds, conceal etc. 

We have plans to enrich table support in neorg like conceal the whole thing with the visual representation with ascii arts, or make a command that launches a big floating window to _edit_ the table and accepts keybinds such as `<Tab>, <CR>` just like the spreadsheet programs. 

But (a very big but) this is very far in the todo list and no one has even started working on it. Unfortunately there are some tasks blocking table feature (like the new v3 parser) that has to be done before start working on this. 

IMHO I use bullet list which does a good enough job for now. 
And `@table` when I really need to.
